scripts/generate_syncset.py

```diff
@@ -29,31 +30,62 @@ def get_all_yaml_obj(file_paths):
             yaml_objs.append(obj)
     return yaml_objs
 
+def process_yamls(name, directory, obj):
+    o = copy.deepcopy(obj)
+    # Get all yaml files as array of yaml objecys
+    yamls = get_all_yaml_obj(get_all_yaml_files(directory))
+
+    for y in yamls:
+        if 'patch' in y:
+            if not 'patches' in o['spec']:
+                o['spec']['patches'] = []
+            o['spec']['patches'].append(y)
+        else:
+            if not 'resources' in o['spec']:
+                o['spec']['resources'] = []
+            o['spec']['resources'].append(y)
+
+    o['metadata']['name'] = name
+
+    # append object to the template's objects
+    template_data['objects'].append(o)
+
 
 if __name__ == '__main__':
     #Argument parser
     parser = argparse.ArgumentParser(description="selectorsyncset generation tool", usage='%(prog)s [options]')
     parser.add_argument("--template-path", "-t", required=True, help="Path to template file [required]")
     parser.add_argument("--yaml-directory", "-y", required=True, help="Path to folder containing yaml files [required]")
     parser.add_argument("--destination", "-d", required=True, help="Destination for selectorsynceset file [required]")
-    parser.add_argument("--git-hash", "-c", required=True, help="Commit hash of commit used [required]")
+    parser.add_argument("--repo-name", "-r", required=True, help="Name of the repository [required]")
     arguments = parser.parse_args()
 
-    # Get all yaml files as array of yaml objecys
-    yamls = get_all_yaml_obj(get_all_yaml_files(arguments.yaml_directory))
     # Get the template data
     template_data = get_yaml(arguments.template_path)
-    # Configure template
+    # Configure template.  Use the last object as a template for adding SSS for each child dir of yaml_directory.
```

Comment:
> `Use the last object as a template for adding SSS for each child dir of yaml_directory.`

Why? Can you expand on why this decision is made / why it's important?

Reply:
Instead of having a separate template for what the SSS looks like I am simply taking the last element of the objects array.  Probably better to have a separate file for this. I can make that change..

Reply:
Updated to have separate template files for the `Template` and `SelectorSyncSet`.